src/m365/spo/commands/file/file-rename.ts

```diff
@@ -0,0 +1,127 @@
+import { Cli, Logger } from '../../../../cli';
+import Command, {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { urlUtil, validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { Options as SpoFileRemoveOptions } from './file-remove';
+const removeCommand: Command = require('./file-remove');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  sourceUrl: string;
+  targetFileName: string;
+  force?: boolean;
+}
+
+class SpoFileRenameCommand extends SpoCommand {
+  public get name(): string {
+    return commands.FILE_RENAME;
+  }
+
+  public get description(): string {
+    return 'Renames a file';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.force = !!args.options.force;
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const webUrl = args.options.webUrl;
+    const originalFileServerRelativeUrl: string = urlUtil.getServerRelativePath(args.options.webUrl, args.options.sourceUrl);
+    
+    this
+      .fileExists(originalFileServerRelativeUrl, webUrl)
+      .then((): Promise<void> => {
+        if (args.options.force) {
```

Comment:
I noticed this strange behavior that when I have source file blocked (so will resolve in error) and I don't use the `--force` option the error message is fine. But when I use the `--force` I get the object as result 🤔
```
PS C:\Workspace\CLI\cli-microsoft365> m365 spo file rename --webUrl https://tenanttocheck.sharepoint.com/sites/hr-life --sourceUrl '/Shared Documents/Document.docx' --targetFileName 'Test1.docx' --force
Error: [object Object]
PS C:\Workspace\CLI\cli-microsoft365> m365 spo file rename --webUrl https://tenanttocheck.sharepoint.com/sites/hr-life --sourceUrl '/Shared Documents/Document.docx' --targetFileName 'Test1.docx'
Error: The file "https://tenanttocheck.sharepoint.com/sites/hr-life/Shared Documents/Document.docx" is locked for shared use by adam@tenanttocheck.onmicrosoft.com.
```
would you be so kind to have a look on that🙏?
let me know if you need any support 😊